package Exceptions;

import java.util.Scanner;

public class Exceptions
{
    public static void main(String[] args)
    {
        System.out.println("program is started...");

        Scanner in = new Scanner(System.in);
        //Eg:1
//        System.out.println("Enter a number");
//        int num = in.nextInt();
//        System.out.println(100/num);         //Arithmetic Exception

        //Eg:2
//        int a[] = new int[5];
//        System.out.println("Enter the position (0-4)");
//        int pos=in.nextInt();
//
//        System.out.println("Enter the value");
//        int value=in.nextInt();
//
//        a[pos]=value;         //ArrayIndexOutOfBoundsException
//
//        System.out.println(a[pos]);

          //Eg:3
//        String s="welcome";
//        int num1=Integer.parseInt(s);  //NumberFormatException
//        System.out.println(num1);

        //Eg:4
//        String s=null;
//        System.out.println(s.length()); //NullpointerException


        System.out.println("program is completed...");
        System.out.println("program is exited...");



    }
}
